### PR TITLE
feat: project connect-tenant

### PR DESCRIPTION
**Pull request by benPearce1**
[sc-27715]

Simply a project command wrapping the `tenant connect` command
 
image

image

image

### CODE REVIEW

**Comment by jbristowe** [comment]
I think we should name this `octopus project connect`. Conversely, we could also have a `octopus project disconnect`.

File: pkg/cmd/project/connect-tenant/connect-tenant.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+func NewCmdConnect(f factory.Factory) *cobra.Command {
+	connectFlags := connectTenant.NewConnectFlags()
+	cmd := &cobra.Command{
+		Use:   "connect",
+		Short: "Connect a tenant to a project in Octopus Deploy",
+		Long:  "Connect a tenant to a project in Octopus Deploy",
+		Example: fmt.Sprintf(heredoc.Doc(`
+			$ %s project connect
+			$ %s project connect --project "Deploy web site" --environment "Production"
```
Comment (jbristowe) [inline pkg/cmd/project/connect-tenant/connect-tenant.go:21]: Is this right? Should it be `--tenant`?

Comment (domenicsim1) [reply pkg/cmd/project/connect-tenant/connect-tenant.go:21]: Yes, this is correct but I think we should have an example that shows specifying both project and tenant.

**Review by domenicsim1 — APPROVED** [review]

File: pkg/cmd/project/connect-tenant/connect-tenant.go (hunk 1 above)
Comment (jbristowe) [reply pkg/cmd/project/connect-tenant/connect-tenant.go:21]: This should read, `%s project connect --tenant "Test Tenant" --environment "Production"`

Comment (benPearce1) [reply pkg/cmd/project/connect-tenant/connect-tenant.go]: updated both commands to include `--tenant` in the example

**Pull request merged**